packages/tooling/fast-vscode-syntax-highlighting/syntaxes/fast-tagged-templates.tmLanguage.json

```diff
@@ -15,7 +15,7 @@
     ],
     "repository": {
         "css-tagged-template": {
-            "begin": "(?i)(?<!\\.)(css|((?:\\/\\*\\s*?)css(?:\\s*\\*\\/)))\\s*(`)",
+            "begin": "(?i)(?<!\\.)((cssPartial|css)|((?:\\/\\*\\s*?)(cssPartial|css)(?:\\s*\\*\\/)))\\s*(`)",
```

Comment:
The syntax for this has changed for the in development versions of FAST so this should be updated to handle that as well. `cssPartial` was deprecated and added as an extension to `css` in the form of `css.partial`